src/en/clean-copy/02-Part I. From Sumerians to Sadi Carnot/12.md

```diff
@@ -2,52 +2,52 @@
+Tasting a ‘real’ IPA, the one that Hodgson and Allsopp shipped to India, is unfortunately not possible. To begin with, we have no idea what they were actually shipping. Out of four original East India Company suppliers (namely, Hodgson, Allsopp, Bass, and Salt), none have survived. The Bass trademark was bought by AB InBev, and Bass Pale Ale continues to be produced (under the pretentious ‘Bass Trademark No. 1 Ale’ name) with the red triangle being redesigned beyond any recognition. However, it has nothing to do with the original Bass, and furthermore, the beer itself is quite mediocre.
 
-Craft brewers make some aged intensively hopped stock ales that should be very much like the real IPAs. That's for example, Brett IPA by Allagash and Enjoy After Brett IPA by Stone.
+Todays's craft brewers make some extensively hopped stock ales that should be very much like the real IPAs. For example, there are Brett IPA by Allagash and Enjoy After Brett IPA by Stone.
 
 #### Bitterness Units
 
-Bitterness of beer might be measured. There are two main scales: international (IBU, International Bitterness Units) and European (EBU, European Bitterness Units). Beer with less than 30 IBUs is usually considered not bitter at all (though an English man of the 15th century wouldn't agree with that). Beer with 60 or more IBUs is definitely bitter, and many people will not like it (pale ale at this level of bitterness should be called an IPA). In the 30 to 60 range, every producer is free to decide whether to mark the beer as a pale ale, a bitter, or maybe an IPA as well.
+Bitterness of beer can be definitively measured. There are two main scales: international (IBU, International Bitterness Units) and European (EBU, European Bitterness Units). Beer with less than 30 IBUs is usually considered not bitter at all (though an English man of the 15th century wouldn't agree with that). Beer with 60 or more IBUs is definitely bitter, and many people will not like it (pale ale at this level of bitterness should be called an IPA). In the 30 to 60 range, every producer is free to decide whether to mark the beer as a pale ale, a bitter, an ESB, or maybe an IPA as well.
 
-Thicker and stronger beers conceal bitterness. Imperial stouts (see the next chapter) might easily have IBU 100, but it's hard to tell they did. Reaching more than IBU 100 is possible but makes no sense as humans can't sense it.
+Thicker and stronger beers conceal bitterness. Imperial stouts (see the next chapter) might easily have 100 IBUs, but it's hard to tell for sure. Reaching more than 100 IBUs is possible but makes no sense as humans can't sense it.
```

Comment:
My intent was: they have 100 IBUs, but you hardly can sense it in their taste as it's overpowered with other flavors. Maybe a better wording is needed